tests/ut/GstreamerMseAudioSinkTests.cpp

```diff
@@ -434,6 +435,102 @@ TEST_F(GstreamerMseAudioSinkTests, ShouldGetStreamSyncModeProperty)
     gst_object_unref(textContext.m_pipeline);
 }
 
+TEST_F(GstreamerMseAudioSinkTests, ShouldReturnDefaultFadeVolumeValueWhenPipelineIsBelowPausedState)
+{
+    RialtoMSEBaseSink *audioSink = createAudioSink();
+
+    guint fadeVolume{0};
```

Comment:
0 is the default, set it as a value that isnt 0 so we check that it gets set correctly by gstreamer